cypress/e2e/signIn.cy.js

```diff
@@ -18,21 +18,30 @@ describe('Sign In page', () => {
   });
 
   it('should provide an ability to log in with existing credentials', () => {
+    cy.register();
     signInPage.visit();
-    cy.register(user.email, user.username, user.password);
-
+    
     signInPage.emailField
-      .type(user.email);
+      .type('riot@qa.team');
```

Comment:
It's recommended to use variables instead of hardcoded values. This makes the code easier to manage and modify. In this case, you should use 'user.email' instead of the hardcoded string 'riot@qa.team'.